client/my-sites/media-library/content.jsx

```diff
@@ -59,6 +60,40 @@ class MediaLibraryContent extends React.Component {
 		source: '',
 	};
 
+	constructor( props ) {
+		super( props );
+
+		this.state = { folders: [] };
+
+		this.parseFolders = this.parseFolders.bind( this );
```

Comment:
Let's get rid of these `bind`s and change the function definitions, e.g. `parseFolders = ( { media } ) => {`

Reply:
Fixed as of [this commit](https://github.com/Automattic/wp-calypso/pull/25763/commits/bb404944bde001be1789cbdccebf1ebc811c223f)